eval/list_test.go

```diff
@@ -0,0 +1,133 @@
+package eval
+
+import "testing"
+
+func TestParseAndFixListIndex(t *testing.T) {
+	check := func(t *testing.T, name, value string, want, got interface{}) {
```

Comment:
You can remove the `t` parameter as `check` is a closure. Consider renaming this function to `checkEqual`.